src/styles/vars-and-mixins.scss

```diff
@@ -550,4 +550,10 @@ $default-box-padding: 1.5rem 2rem;
     white-space: pre;
     text-align: left;
   }
+}
+
+@mixin text-ellipsis() {
```

Comment:
There is already such mixin in `@import 'arachne-ui-components/lib/styles/vars-and-mixins.scss';`